Userland/Libraries/LibJS/Runtime/AbstractOperations.h

```diff
@@ -90,4 +92,14 @@ auto modulo(T x, U y) requires(IsArithmetic<T>, IsArithmetic<U>)
     }
 }
 
+template<typename T, typename U>
+auto modulo(T const& x, U const& y) requires((IsSame<RemoveCVReference<T>, Crypto::SignedBigInteger> || IsSame<RemoveCVReference<T>, Crypto::UnsignedBigInteger>), (IsSame<RemoveCVReference<U>, Crypto::SignedBigInteger> || IsSame<RemoveCVReference<U>, Crypto::UnsignedBigInteger>))
```

Comment:
Might be worth it to define this in LibCrypto as `IsBigInteger` to avoid repeating yourself

Reply:
Also if it is made into a separate concept, you can then simplify this with:
```c++
template<BigInteger T, BigInteger U>
auto modulo(T const& x, U const& y)
{
}
```

Reply:
or even
```c++
auto modulo(BigInteger const& x, BigInteger const& y)
{
}
```

Reply:
Can you use concepts as non-template parameter types?

This:
```c++
template<typename T>
concept BigInteger = IsSame<RemoveCVReference<T>, Crypto::SignedBigInteger> || IsSame<RemoveCVReference<T>, Crypto::UnsignedBigInteger>;

void modulo(BigInteger const&, BigInteger const&)
{
}
```

Gives me this error:
```
error: expected ‘auto’ or ‘decltype(auto)’ after ‘BigInteger’
  135 | void modulo(BigInteger const&, BigInteger const&)
      |             ^~~~~~~~~~
error: expected ‘auto’ or ‘decltype(auto)’ after ‘BigInteger’
  135 | void modulo(BigInteger const&, BigInteger const&)
      |                                ^~~~~~~~~~
```

Reply:
you can, I just forgot to include the `auto` as the error suggests (auto arguments are implicit typename template parameters, and if you prepend a concept it is an implicit concept template parameter)

Reply:
Neat, TIL!
"Abbreviated function template" - https://en.cppreference.com/w/cpp/language/function_template

Reply:
TIL as well, thanks for the suggestions!